pkg/api/v1/types.go

```diff
@@ -3089,6 +3089,13 @@ type Secret struct {
 	// Described in https://tools.ietf.org/html/rfc4648#section-4
 	Data map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
 
+	// StringData allows specifying non-binary secret data in string form.
+	// It is provided as a write-only convenience method.
+	// All keys and values are moved to the data field on write.
```

Comment:
I meant "update the comment" with that info :)

On Wed, Jun 29, 2016 at 8:28 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/api/v1/types.go
> <https://github.com/kubernetes/kubernetes/pull/28263#discussion_r69064660>
> :
>
> > @@ -3089,6 +3089,13 @@ type Secret struct {
> >  	// Described in https://tools.ietf.org/html/rfc4648#section-4
> >  	Data map[string][]byte `json:"data,omitempty" protobuf:"bytes,2,rep,name=data"`
> >
> > +	// StringData allows specifying non-binary secret data in string form.
> > +	// It is provided as a write-only convenience method.
> > +	// All keys and values are moved to the data field on write.
>
> yeah, merged. stringData wins... it's write-only, so any time you specify
> it, you're saying you want it. that lets patch work as intended (you can
> patch a field in either map and it takes effect)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28263/files/ca0c1ab6efc5ab4ed34d3fcdeb5c126007fb4320#r69064660>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVPALGQmILLBCiC46qquBldHsmJZLks5qQzfhgaJpZM4JBwt9>
> .
>